uniffi_bindgen/src/bindings/python/gen_python.rs

```diff
@@ -206,6 +230,21 @@ mod filters {
             Type::Boolean => format!("(True if {} else False)", nm),
             Type::String => format!("{}.consumeIntoString()", nm),
             Type::Object(name) => format!("{}._make_instance_({})", class_name_py(name)?, nm),
+            Type::ExternalType(ext_name, primitive) => {
+                let prim_lift = format!(
```

Comment:
an alternative strategy here might be to generate custom global helpers for each type and this can shrink to identical boilerplate regardless of implementation. That's just shifting the code to a more dedicated "setup" step, but that might be worthwhile.